node_modules/gh-groups/lib/rest.js

```diff
@@ -23,6 +23,32 @@ var GrassHopper = require('gh-core');
 var GroupsAPI = require('./api');
 
 /**
+ * @REST getUserMemberships
+ *
+ * Get the groups a user is a member of
+ *
+ * @Server      admin,app
+ * @Method      GET
+ * @Path        /users/{id}/memberships
+ * @PathParam   {number}            id                      The id of the user to get the upcoming events for
+ * @QueryParam  {number}            [limit]                 The maximum number of results to retrieve. Default: 25
+ * @QueryParam  {number}            [offset]                The paging number of the results to retrieve
+ * @Return      {EventList}                                 The upcoming events for the user
+ */
+var getUserMemberships = function(req, res) {
```

Comment:
I don't really think there is a way to automate this process, so each one we add is just adding more work. I'd prefer if we just change them incrementally as we come across them / touch the files.